agent/conf/agent.properties

```diff
@@ -202,6 +202,11 @@ hypervisor.type=kvm
 # Disable memory ballooning on vm guests for overcommit, by default overcommit
 # feature enables balloon and sets currentMemory to a minimum value.
 #
+# The time interval (in seconds) at which the balloon driver will get memory stats updates.
+# This is equivalent to Libvirt's --period parameter when using the dommemstat command.
+# Default value: 0
```

Comment:
As the commented property already has the default value, the line `# Default value: 0` is not necessary.

Reply:
Done.